owasp-suppressions.xml

```diff
@@ -8,7 +8,7 @@
     <cpe>cpe:/a:grpc:grpc</cpe>
     <cpe>cpe:/a:utils_project:utils</cpe>
   </suppress>
-  <suppress until="2023-12-31Z">
```

Comment:
These suppression entries can just be removed, the dependencies should be up to date already (once you merge main).